javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/Java8LambdaTransformer.java

```diff
@@ -5,31 +5,26 @@
 
 package io.opentelemetry.javaagent.tooling;
 
+import io.opentelemetry.javaagent.bootstrap.LambdaTransformer;
 import java.lang.instrument.ClassFileTransformer;
 import java.lang.instrument.IllegalClassFormatException;
-import java.security.ProtectionDomain;
 
-/** {@link ClassFileTransformer} for lambda instrumentation without jpms modules */
-public class LambdaClassFileTransformer implements ClassFileTransformer {
+/** lambda transformer for java < 9 without jpms modules support */
+public class Java8LambdaTransformer implements LambdaTransformer {
 
   private final ClassFileTransformer delegate;
 
-  public LambdaClassFileTransformer(ClassFileTransformer delegate) {
+  public Java8LambdaTransformer(ClassFileTransformer delegate) {
     this.delegate = delegate;
   }
 
   @Override
-  public byte[] transform(
-      ClassLoader loader,
-      String className,
-      Class<?> classBeingRedefined,
-      ProtectionDomain protectionDomain,
-      byte[] classfileBuffer)
+  public byte[] transform(String className, Class<?> targetClass, byte[] classfileBuffer)
       throws IllegalClassFormatException {
 
     // lambda instrumentation happens only when the lambda is defined, thus the classBeingRedefined
```

Comment:
should be `targetClass` instead of `classBeingRedefined`?

Reply:
I'm not sure to get what you mean here, can you clarify a bit ?